sdk/nodejs/resource.ts

```diff
@@ -448,6 +449,25 @@ export interface ResourceOptions {
      * If this is a [ComponentResourceOptions] do not provide both [provider] and [providers]
      */
     provider?: ProviderResource;
+    /**
+     * An optional customTimeouts configuration block.
+     */
+    customTimeouts?: CustomTimeouts;
+}
+
+export interface CustomTimeouts {
+    /**
+     * The optional create timeout represented as a string e.g. 5m.
```

Comment:
Might be good to have additional examples (e.g. `5m, 40s, 1d`, etc.). I'd almost suggest linking to the Go docs, but that feels a little odd.

Reply:
Good idea!